### ISSUE TITLE
Docs 'Getting Started' page not correct

### ISSUE THREAD

**Issue by rdcoe** [elastic/logstash#2934]
On the getting-started-with-logstash, your docs say to run this:
bin/logstash -e 'input { stdin { } } output { stdout {} }'

What's missing is the "agent" in the command-line.

Besides that, if you try and run ".\bin\logstash.bat agent -e" the help message is output.  The help clearly states that the default behaviour for -e is an empty string, which should pipe stdin/stdout.

If you run "bin\logstash agent -e 'input { stdin { } } output { stdout {} }'" in a Windows cmd shell, then you're greeted with
No such command "{"
Usage: logstash <command> [command args]

And in PowerShell:
bin\logstash agent -e 'input { stdin { } } output { stdout {} }'
{ was unexpected at this time.

In Cygwin (with Ruby sdk and debug tools installed):
$ bin/logstash agent -e 'input { stdin { } } output { stdout {}
}'
Error: Unable to access jarfile /cygdrive/c/Users/rcoe/logstash-1.4.2/vendor/jar/jruby-complete-1.7.11.jar

And finally, when I run on an Ubuntu 14-04LTS openstack VM, I run the command and it seems to run, but when I type 'hello' into another another console, I get nothing in the shell I invoked the command from (see screen shot).
![logstash](https://cloud.githubusercontent.com/assets/11742613/6949375/06a08b1e-d882-11e4-8a05-3f5f13ca246c.png)

In all cases, I am running Java 1.8 (1.8.0_31-b13 on Windows and 1.8.0_40-b25 on unbuntu).  I am running logstash 1.4.2.  I am also importign the TabExpansion++ module in PowerShell but cmd doesn't use that, so not sure how to escape the { so it gets through to the Ruby interpreter.  I also can't find the monolithic jar that used to exist.

Any thoughts?  Seems pretty strange that nothing I try will run this.  I'm guessing it's Java 8 but I haven't gone down the route of installing out-of-date versions.

Thanks.

**Comment by untergeek** [comment]
On which site are you looking at the doc with the mistake?

Yes, in 1.4.2, it's bin\logstash agent, but that's being done away for 1.5.

**Comment by rdcoe** [comment]
http://logstash.net/docs/1.4.2/tutorials/getting-started-with-logstash

**Comment by untergeek** [comment]
Apologies, but we're not likely to fix the 1.4.2 documentation as 1.5 is soon at hand. The 1.5 documentation is at http://www.elastic.co/guide/en/logstash/current/logstash-reference.html.  As stated, with Logstash 1.5 the `agent` option is no longer needed in the command-line.

If you find errors with the 1.5 documentation, please let us know.

**Comment by magnusbaeck** [comment]
> If you run "bin\logstash agent -e 'input { stdin { } } output { stdout {} }'" in a Windows cmd shell, then you're greeted with
No such command "{"
Usage: logstash [command args]

I believe you'll have better luck using double quotes instead of single quotes.

> And finally, when I run on an Ubuntu 14-04LTS openstack VM, I run the command and it seems to run, but when I type 'hello' into another another console, I get nothing in the shell I invoked the command from (see screen shot).

This is completely expected. What you type in one shell does not propagate to other shells.

**Comment by rdcoe** [comment]
I should have mentioned that I tried various quotes and escapes and nothing
worked in my Windows shells.  I also realized my error about using
different shells to communicate through stdin before I sent this report in,
but in my frustrations to get it to work, I forgot about that.  I had
earlier tried backgrounding the process and using the same shell to pipe
stdin to logstash, but did not have success.

With the various different behaviours surrounding logstash on different
platforms and shells, I'm surprised that I haven't seen more reports of
similar experiences.  This is my dev machine, running Windows7 Enterprise,
so perhaps there is some side effect introduced into my shells by .NET
libraries or such?  I normally develop in Linux, but these errors,
especially the cygwin error about not finding the jar, are completely new
to me.  Doesn't give me a lot of confidence that I could write a Windows
installer that would guarantee success.
